JamGame/Assets/Scripts/LevelManagement/ConfigFiles/LevelConfig.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class Tariffs
+{
+    public float WaterCost;
+    public float ElectricityCost;
+}
+
+[Serializable]
+[CreateAssetMenu(fileName = "LevelConfig", menuName = "Level/LevelConfig", order = 0)]
+public class LevelConfig : ScriptableObject
+{
+    public float BossStressSpeed;
+    public Tariffs Tariffs;
+    public List<DayConfig> Days;
+    public DayConfig DefaultDay;
+}
```

Comment:
Make all the fields readonly (`List<DayConfig>` too, see https://github.com/mertwole/miniature-happiness/blob/f7d204c6068b0e6f7c4d101bb9c684a375c3a5e4/JamGame/Assets/Scripts/Employee/Buff/Effect/NeedModifierEffect.cs#L10)